apps/namadillo/src/atoms/accounts/services.ts

```diff
@@ -38,3 +38,12 @@ export const fetchAccountBalance = async (
       BigNumber(balance.amount).shiftedBy(-decimals)
     : BigNumber(0);
 };
+
+export const fetchAccountBalance = async (
+  api: DefaultApi,
+  account: Account | undefined
+): Promise<Balance[]> => {
+  if (!account) return [];
+  const balancesResponse = await api.apiV1AccountAddressGet(account.address);
```

Comment:
@emccorson I didn't update here because the current `namadaAsset.base` is `namnam`, and it'll create a cascade of issues. In case you prefer to change the `namadaAsset.base` to be `nam`, I can shift this value returned from the indexer api :)